README.md

```diff
@@ -9,3 +9,37 @@ Package for email notifiations with editable templates (for important user-relat
 [![downloads](https://img.shields.io/packagist/dt/escolalms/templates-email)](https://packagist.org/packages/escolalms/templates-email)
 [![downloads](https://img.shields.io/packagist/v/escolalms/templates-email)](https://packagist.org/packages/escolalms/templates-email)
 [![downloads](https://img.shields.io/packagist/l/escolalms/templates-email)](https://packagist.org/packages/escolalms/templates-email)
+
+## What does it do
+
+This package allows you to add template email includes event and email variable
```

Comment:
This package allows you to create email notifications for events emitted in Laravel app. 

Each notification is created as a class describing available variables that can be used in Template (which will be created in database and editable through admin panel). This class must be registered using Template facade from Template package, where you specify which Event it is associated with and which Channel it is sent through (e.g. Email channel when defined in this package).